code/controllers/subsystem/id_access.dm

```diff
@@ -123,6 +127,16 @@ SUBSYSTEM_DEF(id_access)
 	accesses_by_region[REGION_CENTCOM] = REGION_ACCESS_CENTCOM
 
 	station_regions = REGION_AREA_STATION
+	remotes_listening_by_region = list(
+		REGION_ALL_STATION = list(),
+		REGION_SECURITY = list(),
+		REGION_MEDBAY = list(),
+		REGION_RESEARCH = list(),
+		REGION_ENGINEERING = list(),
+		REGION_SUPPLY = list(),
+		REGION_COMMAND = list(),
+		REGION_GENERAL = list(),
+	)
```

Comment:
Why prefill it with keys? Why not make it completely lazy?

Reply:
I'm still in the stages of everything working as intended, I'll be going back over to clean up